nix/pkgs/mantis/unwrapped.nix

```diff
@@ -0,0 +1,51 @@
+# this file originates from SBTix
```

Comment:
can we improve the comment, i mean, this file is originates from SBTix but as far as i know we also added some stuff.